third_party/terraform/tests/resource_compute_instance_template_test.go.erb

```diff
@@ -2026,6 +2069,43 @@ resource "google_compute_instance_template" "foobar" {
 `, suffix, enableSecureBoot, enableVtpm, enableIntegrityMonitoring)
 }
 
+<% unless version == 'ga' -%>
+func testAccComputeInstanceTemplateConfidentialInstanceConfig(suffix string, enableConfidentialCompute bool) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  provider = google-beta
+  family  = "ubuntu-2004-lts"
+  project = "ubuntu-os-cloud"
+}
+
+resource "google_compute_instance_template" "foobar" {
+  name         = "cvm-%s"
```

Comment:
You'll want to make sure you're using the `google-beta` provider here. Also, update the fields to correspond with the instance_template rather than the instance.